server/routes/viewingReports/statements.test.ts

```diff
@@ -142,7 +142,7 @@ describe('GET /:reportId/add-comment-to-statement', () => {
       .expect(200)
       .expect(res => {
         expect(res.text).toContain('Add a comment to your statement')
-        expect(res.text).toContain(1)
+        expect(res.text).toContain('1')
```

Comment:
Not sure if I was right to change this number to a string but the test wanted it.